platform/darwin/src/MGLTilePyramidOfflineRegion.mm

```diff
@@ -29,12 +29,13 @@ -(NSDictionary *)offlineStartEventAttributes {
              MMEEventKeyShapeForOfflineRegion: @"tileregion",
              MMEEventKeyMinZoomLevel: @(self.minimumZoomLevel),
              MMEEventKeyMaxZoomLevel: @(self.maximumZoomLevel),
-             MMEEventKeyStyleURL: self.styleURL
+             MMEEventKeyStyleURL: self.styleURL.absoluteString ?: [NSNull null]
              #endif
              };
 }
 
 + (BOOL)supportsSecureCoding {
+    
```

Comment:
nit: extra space here (may not be a concern)

Reply:
Whoops - that's a typo.